test/functional/wallet_importmulti.py

```diff
@@ -340,8 +343,7 @@ def run_test(self):
                                "timestamp": "now",
                                "redeemscript": multisig.redeem_script},
                               True)
-        address_assert = self.nodes[1].getaddressinfo(multisig.p2sh_addr)
-        assert_equal(address_assert['timestamp'], timestamp)
+        self.test_address(multisig.p2sh_addr, timestamp=timestamp)
```

Comment:
IMO it wouldn't hurt to test for solvability here too to ensure the same output as listunspent below